src/dataframerow/dataframerow.jl

```diff
@@ -32,12 +32,16 @@ row(r::DataFrameRow) = getfield(r, :row)
 
 Base.view(adf::AbstractDataFrame, rowind::Integer, ::Colon) =
     DataFrameRow(adf, rowind)
+Base.view(adf::SubDataFrame, rowind::Integer, ::Colon) =
+    DataFrameRow(parent(adf), rowind)
```

Comment:
The index is incorrect AFAICT. Also needs a test then.

Reply:
Thank you. Good catch. Actually I was aware of this, but forgotten to fix 😭.

Moreover I have written a test for it, but I was silly enough to make a `SubDataFrame` from a `DataFrame` starting from take the first row, when actually this passes OK.

I will fix all the tests to be better.

Reply:
a similar bug is also below.